spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -43,17 +47,22 @@
     end
   end
 
-  context 'email_analytics_attributes' do
-    before(:each) do
-      stub_request_passed_proofing_results
-    end
-
-    it 'logs message with email analytics attributes' do
-      freeze_time do
-        job.perform(Time.zone.now)
+  it 'logs message with email analytics attributes' do
+    freeze_time do
+      job.perform(Time.zone.now)
+      if email_type == 'deadline passed'
+        expect(job_analytics).to have_logged_event(
+          'GetUspsProofingResultsJob: deadline passed email initiated',
+          enrollment_code: pending_enrollment.enrollment_code,
+          enrollment_id: pending_enrollment.id,
+          wait_until: anything,
+          service_provider: pending_enrollment.issuer,
+          timestamp: Time.zone.now,
+        )
+      else
         expect(job_analytics).to have_logged_event(
           'GetUspsProofingResultsJob: Success or failure email initiated',
-          email_type: anything,
+          email_type: email_type,
```

Comment:
now that we're no longer always sending 'passed' emails, we can expect a specific `email_type`